app/code/core/Mage/Adminhtml/Block/Sales/Order/Create/Search.php

```diff
@@ -16,10 +17,25 @@
  */
 class Mage_Adminhtml_Block_Sales_Order_Create_Search extends Mage_Adminhtml_Block_Sales_Order_Create_Abstract
 {
+    /**
+     * Contains button descriptions to be shown at the top of accordion
+     * @var list<array>
+     */
+    protected $_buttons = [];
```

Comment:
In this class I didn't add native PHP types to methods and properties and I used the underscore.

If we add them, then this block would work differently than the other `Mage_Adminhtml_Block_Sales_Order_Create_*` blocks, and I'm not sure that's great.

Any thoughts?

Reply:
mmm tricky situation but I think coherence makes more sense in this case, so I'd live it as you implement it already. If in the future we mass add types then at least all those classes will still be similar between each other.

Reply:
> If in the future we mass add types then at least all those classes will still be similar between each other.

That's what I was thinking too. Maybe someday... :)

Reply:
maybe someday